tests/sentry/receivers/test_releases.py

```diff
@@ -134,6 +148,8 @@ def test_no_matching_group(self):
 
     def test_matching_author_with_assignment(self):
         group = self.create_group()
+        add_group_to_inbox(group, GroupInboxReason.MANUAL)
+        assert GroupInbox.objects.filter(group=group).exists()
```

Comment:
I added assertions in `assertResolvedFromCommit` and `assertNotResolvedFromCommit` that check against GroupIndex. This would fail without the add_group_to_inbox call - but I figure you realized this and just feel the assertion is unnecessary? I removed the assertion :-)